cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/util/Env.java

```diff
@@ -36,6 +40,17 @@ public static String get(@NotNull final String key) {
     }
 
     @NotNull
+    public static Map<String, String> optionsStart(@NotNull final String prefix, final boolean trimPrefix) {
```

Comment:
please rename method to `filterStartsWith(@NotNull final String prefix, final boolean trimPrefix)`.

The name option on line 38 was used to represent an optional value for a given key, but in this scenario we're already returning a 0-many data structure we won't need option in the name.  In actuality we're really filtering all of the environment down to those entries whose key begins with `prefix`.